app/components/classCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, TouchableOpacity } from 'react-native';
+import Colors from '../utils/Colors';
+
+interface ClassCardProps {
+    time: string;
+    title: string;
+    members: string;
+    year: string;
+    status: string;
+    imageUrl: string;
+    onPress: () => void;
+}
+
+const ClassCard = ({ time, title, members, year, status, imageUrl, onPress }: ClassCardProps) => {
+    return (
+        <TouchableOpacity style={styles.card} onPress={onPress}>
+            <Image source={{ uri: imageUrl }} style={styles.image} />
+            <View style={styles.info}>
+                <Text style={styles.time}>{time}</Text>
+                <Text style={styles.title}>{title}</Text>
+                <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+                    <View style={styles.dot} />
+                    <Text style={styles.members}>{members}</Text>
+                </View>
+                <Text style={styles.year}>
+                    <Text style={{ fontWeight: 'bold' }}>Jahrgangsstufe:</Text> {year}
+                </Text>
+                <Text style={styles.status}>
+                    <Text style={{ fontWeight: 'bold' }}>Status: </Text>{status}
+                </Text>
+            </View>
+        </TouchableOpacity>
+    );
+};
+
+const styles = StyleSheet.create({
+    card: {
+        flexDirection: 'row',
+        backgroundColor: Colors.surface,
+        borderRadius: 8,
+        marginVertical: 16,
```

Comment:
Thanks for the updates, just one more thing. Please replace ``marginVertical: 16`` with ``marginBottom: 16``.
Because when you have ``vertical`` it puts the margin on top and bottom and this this that the space between two cards is actually 16+16=32. When we only add it to bottom the space will be 16 from all sides. Then we good!

Reply:
changed all sides to 24 px